paths/intro-to-github/12-merging-pull-requests.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Merging Pull Requests
+permalink: /intro-to-github/12/
+next-page: /intro-to-github/13/
+sidebar:
+  nav: "intro-nav"
+main-content:
+  includes:
+  - activity/intro-to-github/06-merge-a-pr.md
+show-me-how:
+tell-me-why: |
+  ## Merge Explained
+
+  When you merge your branch, you are taking the content and history from the branch you created and adding it to the content and history of the `master` branch.
+
+  ![Merge visual]({{site.baseurl}}/images/merge-visual.jpg)
```

Comment:
@hollenberry I just pushed up a fix. The steps to fix in case you find something similar in the future: 
- Verify the image actually is where it says it is
- Check the path on the link; if it isn't working, look for other working images and emulate that structure. In this case, that went from`{{site.baseurl}}/images/merge-visual.jpg` to `/on-demand/images/merge-visual.jpg`. ✨